src/lib/lodash.ts

```diff
@@ -0,0 +1,55 @@
+export function isUndefined<T>(value: any): value is undefined {
```

Comment:
Not sure if we need a function for that 🤷 We can just write `_ => _ === undefined` wherever

Reply:
We can but if it's inside a filter for example it would not infer the type correctly iwthout the `is undefined` return type. Not sure if we need that in our use case but didn't hurt